build.sh

```diff
@@ -9,6 +9,11 @@ then
 	source /config.env
 fi
 
+if [[ -f /secure/config.env ]]
```

Comment:
I think it might be even better to _fail_ if they don't have a config.env mounted in, there is no reason to ever run this without a custom config.